api/awsauth/README.md

```diff
@@ -18,17 +18,33 @@ Currently we have a `dev` and `prod` stage.
 Deploys are set up to look for environment variables in a `.env` file.
 
 Create a .env file similar to this template:
```

Comment:
instead of trying to manually keep this README in sync with the code how about a little sample as an example of the syntax and a reference to `class EnvConstants` for complete list of variable names?